### ISSUE TITLE
WARNING when webpack loading all *.js.map

### ISSUE THREAD

**Issue by antitoine** [valor-software/ngx-bootstrap#720]
Hi,

After upgrading ng2-bootstrap from 1.0.17 to 1.0.20, a lot of warning appear during the TypeScript compilation : 

```
[...]
WARNING in ./~/ng2-bootstrap/components/rating.js
Cannot find SourceMap 'rating.js.map': Error: Cannot resolve 'file' or 'directory' ./rating.js.map in /var/www/frontend-admin/node_modules/ng2-bootstrap/components

WARNING in ./~/ng2-bootstrap/components/tabs.js
Cannot find SourceMap 'tabs.js.map': Error: Cannot resolve 'file' or 'directory' ./tabs.js.map in /var/www/frontend-admin/node_modules/ng2-bootstrap/components

WARNING in ./~/ng2-bootstrap/components/timepicker.js
Cannot find SourceMap 'timepicker.js.map': Error: Cannot resolve 'file' or 'directory' ./timepicker.js.map in /var/www/frontend-admin/node_modules/ng2-bootstrap/components
[...]
```

Full version here : [warning-bg2-bootstrap.txt](https://github.com/valor-software/ng2-bootstrap/files/358782/warning-bg2-bootstrap.txt)

I'm using webpack (from https://github.com/AngularClass/angular2-webpack-starter).

It seems to work but this is very annoying.

**Comment by Dinistro** [comment]
@antitoine We don't add sourceMaps and webpack seems to require them with your configuration.

**Comment by nick1247** [comment]
Having the same problem.

I don't use bundles, my app is building from required components and ng2-bootstrap components has refs for map files.
'Alert' component for example:

```
...
exports.AlertComponent = AlertComponent;
//# sourceMappingURL=alert.component.js.map
```

**Comment by Dinistro** [comment]
@nick1247 Thanks for this information. Unluckily the sourcemaps get excluded by the .npmignore. 

@valorkin Either we set `"sourceMap": true` in the tsconfig.json or we remove the exclusion in the .npmignore

**Comment by nick1247** [comment]
Thanks

**Comment by fknop** [comment]
Same problem with SystemJS

**Comment by valorkin** [comment]
there are 2 possible options,
- disable source maps for common.js build
- remove source map ignore from npmignore

**Comment by valorkin** [comment]
@antitoine please join https://www.hamsterpad.com/chat/ng2
and drop me a message
so we can try both options fast

**Comment by valorkin** [comment]
source maps removed because they produce warning hell

```
WARNING in ./~/ng2-bootstrap/ng2-bootstrap.js
Cannot find source file 'ng2-bootstrap.ts': Error: Cannot resolve 'file' or 'directory' ./ng2-bootstrap.ts in /home/valorkin/work/open-source/angular2-webpack-starter/node_modules/ng2-bootstrap
```

**Comment by antitoine** [comment]
Fixed with the 1.0.22 npm package ! Thank
